### PR TITLE
[2.19.x] DDF-5888 Checkbox performance improvements (#5886)

### PR DESCRIPTION
**Pull request by jlcsmith**
#### What does this PR do?
Abbreviated review - backport of https://github.com/codice/ddf/pull/5886

#### Who is reviewing it? 

@stevenmalmgren 
@andrewkfiedler 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining 
@andrewkfiedler 

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/row.view.js [hunk 1]
```diff
@@ -33,14 +32,75 @@ module.exports = Marionette.LayoutView.extend({
...
+            <td
+              data-property={`${property.property}`}
+              className={`${property.class} ${
+                property.hidden ? 'is-hidden-column' : ''
+              }`}
+              data-value={`${property.value}`}
+            >
+              <div>
+                {property.value.map(value => {
+                  console.log(value)
```
Comment (bdeining) [inline ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/row.view.js]: lingering log statement

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/thead.view.js [hunk 2]
```diff
@@ -24,11 +23,44 @@ const metacardDefinitions = require('../../singletons/metacard-definitions.js')
...
 const {
-  SelectAllToggle,
-} = require('../../selection-checkbox/selection-checkbox.view.js')
+  ItemCheckboxAll,
+} = require('../../selection-checkbox/item-checkbox-all')
 
 module.exports = Marionette.LayoutView.extend({
-  template,
+  template({ headers }) {
+    console.log(headers)
```
Comment (bdeining) [inline ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/thead.view.js]: lingering log statement

**Comment by bdeining** [comment]
build now

**Comment by jlcsmith** [comment]
build now

**Comment by jlcsmith** [comment]
build now

**Pull request merged**